_data/publications.yml

```diff
@@ -151,10 +151,10 @@
   slug: covid_case_challenges
   authors: Lopez V, Cramer EY, Pagano R, ... Biggerstaff M, Reich NG, Johansson MA
   preprint: https://www.medrxiv.org/content/10.1101/2023.05.30.23290732v1
-  #pdf: https://www.medrxiv.org/content/10.1101/2023.05.30.23290732v1.full.pdf [update when published]
+  pdf: https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011200
   year: 2023
```

Comment:
Can we change this to 2023 and move it up into the "2024" section of the page?

Reply:
Done!